chapters/03_feed-forward.tex

```diff
@@ -285,17 +285,14 @@ \section{Conditionals}\label{sec:ffnn_conditional}
         \end{cases}
     \end{equation*}
 
-    We adapt a construction used in \citep{merrill2024the} (Theorem 1, arXiv version) for two-layer ReLU feedforward nets.
+    We adapt a construction used by \citep{merrill-sabharwal-2024-cot} (Theorem 1) for two-layer ReLU feedforward nets.
```

Comment:
Please use `\citet` and you can put the theorem number in brackets, i.e. `\citet[Theorem 1]{...}`.